src/mono/sample/HelloWorld/Makefile

```diff
@@ -2,7 +2,7 @@ TOP=../../../../
 DOTNET:=$(TOP)./dotnet.sh
 DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
 
-MONO_CONFIG ?=Release
+MONO_CONFIG ?=Debug
```

Comment:
nit: unrelated.  but it also doesn't matter

Reply:
Will address in a following PR.